dev-python/sphinx-panels/sphinx-panels-0.6.0-r1.ebuild

```diff
@@ -22,5 +22,7 @@ RDEPEND="
 
 BDEPEND="test? ( dev-python/pytest-regressions[${PYTHON_USEDEP}] )"
 
+PATCHES=( "${FILESDIR}/${PN}-0.6.0-Make-the-tests-pass-with-Sphinx-7.x.patch" )
```

Comment:
Was this submitted upstream? Please add a link in a comment, so we could track it.

Reply:
Per Debian no:  https://udd.debian.org/patches.cgi?src=sphinx-panels&version=0.6.0-3

But I don't think I can send it, since it's not my patch.

Reply:
You can, Debian does everything on free licenses. Just make sure to mention the original author.

Reply:
And added a note to the patch here.